### PR TITLE
Improve Instant Articles integration page: split into separate components

### PR DESCRIPTION
**Pull request by psiemens**
Today's CPSC 310 lecture inspired me to improve this code. 

Tested with a sample Facebook App. plz merge 😄

### CODE REVIEW

File: dispatch/static/manager/src/js/components/integrations/FBInstantArticles.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+function InstantArticlesPageDropdown(props) {
+  /* Renders a dropdown list to select a Facebook page */
+
+  const pageMap = props.pages.reduce((pageMap, page) => {
+    pageMap[page.id] = page
+    return pageMap
+  }, {})
+
+  const options = props.pages.map(page => {
```
Comment (JimFung) [inline dispatch/static/manager/src/js/components/integrations/FBInstantArticles.js]: probably could have used an implicit return here.

File: dispatch/static/manager/src/js/pages/Integrations/FBInstantArticlesIntegrationPage.js [hunk 2]
```diff
@@ -264,6 +176,9 @@ const mapDispatchToProps = (dispatch) => {
     },
     updateIntegration: (integrationId, data) => {
       dispatch(integrationActions.updateIntegration(integrationId, data))
+    },
+    resetURI: (pathname) => {
```
Comment (JimFung) [inline dispatch/static/manager/src/js/pages/Integrations/FBInstantArticlesIntegrationPage.js:180]: you can make this 1 line case it works without the {} around the dispatch call afaik. you can leave it in if you think it makes it look cleaner/more readable.

**Comment by JimFung** [comment]
the rest LGTM, feel free to merge after changes 👍

File: dispatch/static/manager/src/js/pages/Integrations/FBInstantArticlesIntegrationPage.js (hunk 2 above)
Comment (psiemens) [reply dispatch/static/manager/src/js/pages/Integrations/FBInstantArticlesIntegrationPage.js:180]: yeah i think i prefer it on a separate line

**Comment by psiemens** [comment]
Thanks @JimFung!

**Pull request merged**